### ISSUE TITLE
Intermittent history related crash

### ISSUE THREAD

**Issue by kratostatic** [Idhrendur/paradoxGameConverters#465]
So there's an intermittent crash occuring for at least some users that seems to be related with the history files.  The below is from the forum release thread and is I think the most useful thread on the topic.

Same, after many tests, i found out, that if you try hard enough itll eventually load in for you, which is what i did, i cleared the cache, validated local files, run it without steam, nothing worked, even when editing and not editing the countries in the history folder, with enough attempts, itll eventually work, but the catch is, that itll randomly crash depending on what screen youre on and youre stuck spamming victoria 2 until it doesnt crash, most of my crashes happened, whenever i got through, was when i was messing around in the pop screen. Btw saving the game and reloading it, doesnt work, just crashes the game again, corrupting the save file and if you manage to painstakingly manage to get through the start up loading screens. Any attempt to load the save will crash the game again. It must be something with the converter itself.

EDIT: did more tests, it seems that theres something wrong with the way the converter creates/converts the history files, whenever i copied and paste a fresh set of history files from the output folder in the converter to the victoria mod directory, it would load through but only once as itll keep crashing. The more you edit the history files the more likely it crashes it seems. I also noticed that some provinces didnt get cores, i was playing as spain, and i had land all over the place, yet only having cores on spain, my provinces in portugal, france, africa etc. had no cores. Provinces in china which were culture converted in eu4 200 years prior to the date of conversion, still was completely YUE culture and different religion. There seems to be issues with pops as if you manage to make it through to play the game like i did, the game will crash if you meddle in the pop screen for too long. Even in province view, opening the pop screen will 80%of the time will crash the game. Somehow Austria formed HRE in my latest test which was odd. I would test more to try to help find whats wrong so we can get this converter patched faster (been so keen to continue my campaign), but i have an exam tomorrow.

**Comment by kratostatic** [comment]
Duplicate of #461
